demos/python_demos/segmentation_demo/segmentation_demo.py

```diff
@@ -15,199 +15,216 @@
  limitations under the License.
 """
 
+import logging
+import random
 import sys
-import os
 from argparse import ArgumentParser, SUPPRESS
+from pathlib import Path
+from time import perf_counter
+import os
+
 import cv2
 import numpy as np
-import logging as log
 from openvino.inference_engine import IECore
 
-classes_color_map = [
-    (150, 150, 150),
-    (58, 55, 169),
-    (211, 51, 17),
-    (157, 80, 44),
-    (23, 95, 189),
-    (210, 133, 34),
-    (76, 226, 202),
-    (101, 138, 127),
-    (223, 91, 182),
-    (80, 128, 113),
-    (235, 155, 55),
-    (44, 151, 243),
-    (159, 80, 170),
-    (239, 208, 44),
-    (128, 50, 51),
-    (82, 141, 193),
-    (9, 107, 10),
-    (223, 90, 142),
-    (50, 248, 83),
-    (178, 101, 130),
-    (71, 30, 204)
+sys.path.append(str(Path(__file__).resolve().parents[1] / 'common'))
+
+from models import *
+import monitors
+from pipelines import AsyncPipeline
+from images_capture import open_images_capture
+from performance_metrics import PerformanceMetrics
+
+logging.basicConfig(format='[ %(levelname)s ] %(message)s', level=logging.INFO, stream=sys.stdout)
+log = logging.getLogger()
+
+
+CITYSCAPES_COLORS = [
+    (128, 64, 128),
+    (232, 35, 244),
+    (70, 70, 70),
+    (156, 102, 102),
+    (153, 153, 190),
+    (153, 153, 153),
+    (30, 170, 250),
+    (0, 220, 220),
+    (35, 142, 107),
+    (152, 251, 152),
+    (180, 130, 70),
+    (60, 20, 220),
+    (0, 0, 255),
+    (142, 0, 0),
+    (70, 0, 0),
+    (100, 60, 0),
+    (90, 0, 0),
+    (230, 0, 0),
+    (32, 11, 119),
+    (0, 74, 111),
+    (81, 0, 81)
 ]
 
-np.random.seed(10)
+
+def apply_color_map(input):
+    ### Initializing colors array if needed
+    colors = []
```

Comment:
who should initialize it?

Reply:
Added the `--colors` argument, so the user can initialize colormap as he wishes.